feature-libs/quote/components/config/quote-ui.config.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { Config } from '@spartacus/core';
+import {
+  QuoteActionType,
+  QuoteRoleType,
+  QuoteState,
+} from '@spartacus/quote/root';
+
+export interface QuoteUIConfigFragment {
+  maxCharsForComments?: number;
+  truncateCardTileContentAfterNumChars?: number;
+  confirmActionDialogMapping?: ConfirmActionDialogMappingConfig;
+  updateDebounceTime?: {
+    expiryDate?: number;
+  };
+}
+
+export type ConfirmActionDialogConfig = {
+  i18nKey: string;
```

Comment:
@Uli-Tiger Could we use a better attribute name here? I assume this refers to the success message?

Reply:
actually its a i18nKey prefix used for all configurable text of the the confirm dialog, so
* title
* success message
* warning note
*...
```
    const confirmationContext: ConfirmationContext = {
      quote: quote,
      title: dialogConfig.i18nKey + '.title',
      confirmNote: dialogConfig.i18nKey + '.confirmNote',
    };
    if (dialogConfig.showWarningNote) {
      confirmationContext.warningNote = dialogConfig.i18nKey + '.warningNote';
    }
    if (dialogConfig.showExpirationDate) {
      confirmationContext.validity = 'quote.actions.confirmDialog.validity';
    }
    if (dialogConfig.showSuccessMessage) {
      confirmationContext.successMessage =
        dialogConfig.i18nKey + '.successMessage';
    }
    return confirmationContext;
  }
```
maybe we could name `i18nKeyPrefix`?

Reply:
Yes, that would be an option